perception/autoware_ground_segmentation/src/scan_ground_filter/node.hpp

```diff
@@ -175,21 +175,21 @@ class ScanGroundFilterComponent : public autoware::pointcloud_preprocessor::Filt
   float center_pcl_shift_;  // virtual center of pcl to center mass
 
   // common parameters
-  double radial_divider_angle_rad_;  // distance in rads between dividers
+  float radial_divider_angle_rad_;  // distance in rads between dividers
   size_t radial_dividers_num_;
   VehicleInfo vehicle_info_;
 
   // common thresholds
-  double global_slope_max_angle_rad_;  // radians
-  double local_slope_max_angle_rad_;   // radians
-  double global_slope_max_ratio_;
-  double local_slope_max_ratio_;
-  double split_points_distance_tolerance_;  // distance in meters between concentric divisions
-  double split_points_distance_tolerance_square_;
+  float global_slope_max_angle_rad_;  // radians
+  float local_slope_max_angle_rad_;   // radians
+  float global_slope_max_ratio_;
+  float local_slope_max_ratio_;
+  float split_points_distance_tolerance_;  // distance in meters between concentric divisions
+  float split_points_distance_tolerance_square_;
```

Comment:
I think the primary types of paramters getting through declare_parameter should be double as [here](https://docs.ros.org/en/foxy/Concepts/About-ROS-2-Parameters.html#overview)

Reply:
You also set other parameters in float. Shall I align all the parameters in float?